core/connection/manager_test.go

```diff
@@ -369,15 +370,25 @@ func (fd *fakeDialog) Send(producer communication.MessageProducer) error {
 }
 
 func (fd *fakeDialog) Request(producer communication.RequestProducer) (responsePtr interface{}, err error) {
-	return &session.CreateResponse{
-			Success: true,
-			Message: "Everything is great!",
-			Session: session.SessionDto{
-				ID:     "vpn-connection-id",
-				Config: []byte("{}"),
+	switch producer.(type) {
+	case *promise.Producer:
+		return &promise.Response{
+				Success: true,
+				Message: "Everything is great!",
 			},
-		},
-		nil
+			nil
+	case *session.CreateProducer:
+		return &session.CreateResponse{
+				Success: true,
+				Message: "Everything is great!",
```

Comment:
This field is used for errors only `Success: false`

Reply:
Removed `Message`.